data/city-stats.json

```diff
@@ -49,6 +49,16 @@
     "reforms": "implemented",
     "url": "https://parkingreform.org/mandates-map/city_detail/Anchorage_AK.html"
   },
+  "arcata-ca": {
+    "name": "Arcata, CA",
+    "percentage": "26%",
+    "cityType": "suburb",
+    "population": "18,857",
+    "urbanizedAreaPopulation": "19,714",
```

Comment:
How did you get this number?

Reply:
Submitted along with the maps. Will double check with Tony if we need to double check before.